tests/manage/z_cluster/upgrade/test_check_mon_pdb_post_upgrade.py

```diff
@@ -49,3 +58,22 @@ def test_check_mon_pdb_post_upgrade(self):
             assert (
                 max_unavailable_mon == 1
             ), "Maximum unavailable mon count is not matching"
+
+    @post_ocp_upgrade
+    @post_ocs_upgrade
+    @pytest.mark.polarion_id("OCS-2524")
+    def test_check_osd_pdb_post_upgrade(self):
+        """
+        Test to verify OSD PDBs
+        1. Post OCP and OCS successful upgrades check for OSD PDBs
+        2. Rook-ceph-drain-canary pods disappeared after upgrade of OCS version to ocs-operator.v4.6.2-233.ci
+
+        """
+        assert (
```

Comment:
Should we also consider checking values for disruptions_allowed, min_available_osd, max_unavailable_osd, the way how we are checking for test_check_mon_pdb_post_upgrade?